code/modules/mentor/mentor_verbs.dm

```diff
@@ -12,3 +12,27 @@ GLOBAL_PROTECT(mentor_verbs)
 /client/proc/remove_mentor_verbs()
 	remove_verb(GLOB.mentor_verbs)
 	reset_badges()
+
+/client/verb/mrat()
+	set name = "Request Mentor Assistance"
+	set category = "Mentor"
+
+	if(!istype(src.mob, /mob/living/carbon/human))
```

Comment:
I understand mechanically why Silicons cannot have a Mentor Rat. Being a Brain Trauma and all that. But it probably shouldn't entirely exclude Silicons. I have no idea how that'd work out, but I think the Mentor Rat can be funny and helpful.

Reply:
Please correct me if it does include them, and i've simply missed it.

Reply:
Unfortunately, it does not include Cyborgs/AIs, as first off mechanically, they cannot have brain traumas, which is the mechanic by which this works. Additionally, this feature's primary audience would likely be newer players, and Silicon roles require a decent amount of living hours before being selected. Finally, MRat for an AI would be entirely useless, as it would just be anchored to the AI core, and unable to see what the AI was concerned with.